rooms.yml

```diff
@@ -640,6 +640,7 @@ stackoverflow.com:
       - 72095    # Ryan M
       - 11421497 # cigien
       - 7156005  # Vega
+      - 11355926 # Cow
```

Comment:
The correct network user ID for Cow is 15725245 (see `users.yml`). This file needs the network user, not the local ID. (I too wish all this was less byzantine.)